src/Utilities/generateConfluenceMarkdown.ts

```diff
@@ -15,19 +15,18 @@ export function generateConfluenceMarkdown(
         if (!issues || !issues.length) {
             continue;
         }
-        markdownArray.push('\n');
-        markdownArray.push(`h3. ${dependency}`);
-        markdownArray.push('||*Title*||*Age*||');
-
         issues.forEach((issue) => {
             const createdAt = DateTime.fromISO(issue.createdAt);
             const ageInDays = now.diff(createdAt, 'days').days;
             const ageInWholeDays = Math.round(ageInDays);
 
             if (ageInWholeDays < maxIssuesAge) {
+                markdownArray.push('\n');
```

Comment:
I believe this would repeat the dependency information before each of its issues, which we don't want. 

How could we know in advance (at line 18 in the old code) whether or not this dependency does or does not have any valid issues?